Vagrantfile

```diff
@@ -128,6 +128,13 @@ module Forklift
 
 
   Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+    if Vagrant.has_plugin?("vagrant-hostmanager")
+      config.hostmanager.enabled = true
+      config.hostmanager.manage_host = true
```

Comment:
If anyone wants just the client piece and not manage_host, we can always add some user configuration later.  For now I think these defaults are fine